core/db/migrate/20150716124636_create_spree_shipping_methods_stock_locations.rb

```diff
@@ -0,0 +1,8 @@
+class CreateSpreeShippingMethodsStockLocations < ActiveRecord::Migration
+  def change
+    create_table :spree_shipping_methods_stock_locations do |t|
```

Comment:
ID should be `false` here. It could also just be refactored to:

```ruby
create_join_table :shipping_methods, :stock_locations, table_name: :spree_shipping_method_stock_locations
```